docs/base-spi.rst

```diff
@@ -0,0 +1,8 @@
+Base API
+=====================
+
+.. automodule:: django_spanner.base
+  :members:
+  :inherited-members:
+
+
```

Comment:
nit: is it required to have 2 lines at the end of each file? or will 1 suffice ? wondering if there is a lint rule for this. can you check